tests/fixtures/dag_factory_kubernetes_pod_operator.yml

```diff
@@ -53,7 +53,7 @@ example_dag:
       task_id: 'passing-task'
       get_logs: True
       in_cluster: False
-      dependencies: ['task_1']
+      dependencies: []
```

Comment:
do we need this empty dependency []?

Reply:
Yes, this example had a circular dependency and this was not a valid DAG 🙈